### DISCUSSION TITLE
Moving to more modern package manager

### DISCUSSION THREAD

**Discussion by azzamsa** [almarklein/timetagger#478] [Ideas]
Hi,

Are you open to accepting a pull request that migrates to a more modern package manager?

Instead of using `requirements.txt`, we would transition to using `pyproject.toml`. Perhaps we could consider using something like [rye](https://github.com/astral-sh/rye)

**Comment by almarklein** [comment]
> Instead of using `requirements.txt`, we would transition to using `pyproject.toml`

Yes, would be very open to that.

> Perhaps we could consider using something like [rye](https://github.com/astral-sh/rye)

Using in what sense?
